packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -5739,4 +5741,36 @@ export default class Meeting extends StatelessWebexPlugin {
       this.transcription = undefined;
     }
   };
+
+  /**
+   * enableBNR API
+   * @returns {undefined}
+   * @public
+   * @memberof Meeting
+   */
+  async enableBNR(){
+    LoggerProxy.logger.info('Meeting:index#enableBNR. Enable BNR called');
+    let isSuccess = false;
+    try{
+      if(typeof this.mediaProperties === "undefined" || typeof this.mediaProperties.audioTrack === "undefined"){
+        throw new Error("Meeting doesn't have an audioTrack attached")
+      }
+      this.mediaProperties.audioTrack = await WebRTCMedia.Effects.BNR.enableBNR(this.mediaProperties.audioTrack);
+      const audioStream = new MediaStream();
+      audioStream.addTrack(this.mediaProperties.audioTrack);
+      LoggerProxy.logger.info('Meeting:index#enableBNR. BNR enabled track obtained from WebRTC & sent to updateAudio');
+      await this.updateAudio({
+        sendAudio: true,
+        receiveAudio: true,
+        stream: audioStream
+      });
+      this.isBnrEnabled = true;
+      isSuccess = true;
+    }
+    catch(error){
+      LoggerProxy.logger.error(`Meeting:index#enableBNR. ${error}`);
+      throw error;
+    }
+    return isSuccess;
```

Comment:
Just curious about use of returning a boolean value. Will returning a `Promise.resolve()` solve the purpose here?

Reply:
It's just to indicate the user that BNR is enabled or not. If error is thrown, they'll receive it in the catch block. We cannot resolve the promise in call cases.